tasks/sanity-label-check.yaml

```diff
@@ -23,20 +23,22 @@ spec:
       image: quay.io/redhat-appstudio/hacbs-test:stable
       workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
       script: |
-        skopeo inspect --no-tags docker://$(inputs.params.IMAGE_URL) | tee image_inspect.json
+        if [ -f "$(workspaces.workspace.path)/image_inspect.json" ]; then
```

Comment:
There should be no need for the `if` condition here - if the file is not there, the task will record the error within the `HACBS_ERROR_OUTPUT` value.

Reply:
Hm, we could save the error in an `error.txt` (or similar) file and then add it to the error message like this:
```    
HACBS_ERROR_OUTPUT=$(jq -rc --arg date 1651125649 --arg error "$(cat error.txt)" --null-input \
'{result: "ERROR", timestamp: $date, failures: [$error]}'
```

Reply:
@hongweiliu17 @dirgim could we keep a map of error code and generic error message in the hacbs container and use that to look up generic messages to display in the results output? I could potential  `no such file or directory` being a common error message that would be displayed in an error message we would utilize.

Then the more detailed error message can be redirected to a file?

Reply:
+1

Reply:
> Hm, we could save the error in an `error.txt` (or similar) file and then add it to the error message like this:
> 
> ```
> HACBS_ERROR_OUTPUT=$(jq -rc --arg date 1651125649 --arg error "$(cat error.txt)" --null-input \
> '{result: "ERROR", timestamp: $date, failures: [$error]}'
> ```
> 
> Though we'd have to be careful about the error message size because of the 4k size restriction for Tekton task results.

+1